README.md

```diff
@@ -1,50 +1,30 @@
-[![Build Status](https://github.com/boldlink/<REPO_NAME>/actions/workflows/pre-commit.yml/badge.svg)](https://github.com/boldlink/<REPO_NAME>/actions)
+[![Build Status](https://github.com/boldlink/terraform-github-team/actions/workflows/pre-commit.yml/badge.svg)](https://github.com/boldlink/<REPO_NAME>/actions)
```

Comment:
Change `<REPO_NAME>` to the real repo name. This is in all READMEs in this PR @ndegwajohn